app/move/controllers/view.test.js

```diff
@@ -344,75 +389,259 @@ describe('Move controllers', function () {
-          controller(proposedMoveReq, proposedMoveRes)
-          const params = res.render.args[0][1]
-          expect(params.canCancelMove).to.be.false
+        }
+      })
+
+      context('when record is incomplete', function () {
+        beforeEach(function () {
+          controller(req, res)
+          params = res.render.args[0][1]
+        })
+
+        it('should contain a personEscortRecord', function () {
+          expect(params).to.have.property('personEscortRecord')
+          expect(params.personEscortRecord).to.deep.equal(
+            mockPersonEscortRecord
+          )
+        })
+
+        it('should not show Person Escort Record as complete', function () {
+          expect(params).to.have.property('personEscortRecordIsComplete')
+          expect(params.personEscortRecordIsComplete).to.be.false
+        })
+
+        it('should contain url to Person Escort Record', function () {
+          expect(params).to.have.property('personEscortRecordUrl')
+          expect(params.personEscortRecordUrl).to.equal(
+            `/person-escort-record/${mockPersonEscortRecord.id}`
+          )
+        })
+
+        it('should show Person Escort Record banner', function () {
+          expect(params).to.have.property('showPersonEscortRecordBanner')
+          expect(params.showPersonEscortRecordBanner).to.be.true
+        })
+      })
+
+      context('when record is complete', function () {
+        beforeEach(function () {
+          req.move.profile.person_escort_record = {
+            ...mockPersonEscortRecord,
+            status: 'complete',
+          }
+          controller(req, res)
+          params = res.render.args[0][1]
+        })
+
+        it('should contain a personEscortRecord', function () {
+          expect(params).to.have.property('personEscortRecord')
+          expect(params.personEscortRecord).to.deep.equal({
+            ...mockPersonEscortRecord,
+            status: 'complete',
+          })
+        })
+
+        it('should show Person Escort Record as complete', function () {
+          expect(params).to.have.property('personEscortRecordIsComplete')
+          expect(params.personEscortRecordIsComplete).to.be.true
+        })
+
+        it('should contain url to Person Escort Record', function () {
+          expect(params).to.have.property('personEscortRecordUrl')
+          expect(params.personEscortRecordUrl).to.equal(
+            `/person-escort-record/${mockPersonEscortRecord.id}`
+          )
+        })
+
+        it('should not show Person Escort Record banner', function () {
+          expect(params).to.have.property('showPersonEscortRecordBanner')
+          expect(params.showPersonEscortRecordBanner).to.be.false
+        })
+      })
+
+      context('when feature flag is false', function () {
+        const controllerWithoutPER = proxyquire('./view', {
+          ...pathStubs,
+          '../../../config': {
+            FEATURE_FLAGS: {
+              PERSON_ESCORT_RECORD: false,
```

Comment:
The tests with the feature flag are misleading. The value is never a boolean. It is either a string or undefined.

Reply:
That's actually a good point! I'll update the test for no presence or `false` as a string.